CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.14.2]
+### Added
+* `--publish-bucket` option has been added to the HyP3 entry point to publish product to the ITS_LIVE AWS Open Data bucket, `s3://its-live-data` or the test bucket `s3://its-live-data-test`.
+* `upload_file_to_s3_with_upload_access_keys` to perform S3 uploads with credentialed S3 clients.
```

Comment:
Do users need to know about this function? If not, I suggest deleting this line from the Changelog.
```suggestion
```

Reply:
I think it's relevant. Other members of the ITS_LIVE team will be interested in it.

Reply:
Sounds good.